mongodump/mongodump_test.go

```diff
@@ -12,13 +12,15 @@ import (
 	"crypto/sha1"
 	"encoding/base64"
 	"fmt"
+	"github.com/mongodb/mongo-tools-common/failpoint"
```

Comment:
doesn't look like this line is in the right place. This should be grouped with the other non-stdlib imports